tests/polkadotjs_test/package.json

```diff
@@ -9,7 +9,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@polkadot/api": "4.5.1"
+    "@polkadot/api": "8.8.2"
```

Comment:
😨 that bump!!

Reply:
Yes, it's been a while since we've touched this code.